content-src/components/DiscoveryStreamComponents/TopSites/TopSites.jsx

```diff
@@ -4,14 +4,104 @@
 
 import { connect } from "react-redux";
 import { TopSites as OldTopSites } from "content-src/components/TopSites/TopSites";
+import { TOP_SITES_MAX_SITES_PER_ROW } from "common/Reducers.jsm";
 import React from "react";
 
 export class _TopSites extends React.PureComponent {
+  getNonOrganicSpoc(topSites, data) {
```

Comment:
non-organic spoc could be confusing, my understanding this function filters and help retrieve first available spoc thats not already an organic topsite. Consider renaming to getFirstAvailableSpoc with a comment explaining what it does